crypto/gmac/build.info

```diff
@@ -0,0 +1,2 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=gmac.c
```

Comment:
Oh, a method only implementation?  Now, *that's* rad!  I like!  :smiley:

Reply:
No legacy support this way :)